.github/workflows/deploy-staging.yml

```diff
@@ -0,0 +1,51 @@
+on:
```

Comment:
Will this action actually run on it's on when it has the `on: workflow_call`, but is never called (as far as I could tell)?